internal/bminventory/inventory.go

```diff
@@ -75,6 +75,7 @@ import (
 
 const DefaultUser = "kubeadmin"
 
+const APINotFound = "V1 API not found"
```

Comment:
move this to common so we can use it in other handlers as well. The best place is the common/errors

Reply:
Also please change the wording to something more descriptive like "V1 API is deprecated"

Reply:
I'll move this to common/erros, but this PR is about removal, not deprecation - so the wording looks okay to me.
@avishayt what do you think?

Reply:
:+1: Removal != Deprecation

This looks fine to me. Returning anything more than `Not Found` is more than enough.

Reply:
Agree with @carbonin